src/orion/core/cli/base.py

```diff
@@ -71,6 +72,8 @@ def parse(self, argv):
             format="%(asctime)-15s::%(levelname)s::%(name)s::%(message)s",
             level=levels.get(verbose, logging.DEBUG),
         )
+        if verbose >= 2:
+            print("Orion version : " + orion.core.__version__)
```

Comment:
```suggestion
            logger.debug("Orion version : " + orion.core.__version__)
```
We should use the logger instead and log at DEBUG level. Normally we create the logger at the top of the module so it is widely available (like [here](https://github.com/Epistimio/orion/blob/develop/src/orion/client/experiment.py#L33)). There may be an issue here as we change the configuration with logging.basicConfig. We should make sure that this change is reflected on the configuration of the logger created at the top of this module.